kratos/utilities/csr_conversion_utilities.h

```diff
@@ -0,0 +1,53 @@
+
+//    |  /           | 
+//    ' /   __| _` | __|  _ \   __| 
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/ 
+//                   Multi-Physics  
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//
+
+#if !defined(KRATOS_CSR_CONVERSION_UTILITIES_H_INCLUDED)
+#define  KRATOS_CSR_CONVERSION_UTILITIES_H_INCLUDED
+
+#include "containers/csr_matrix.h"
+
+#include <amgcl/backend/builtin.hpp>
+#include <amgcl/adapter/zero_copy.hpp>
+
+namespace Kratos
+{
+
+/**
+Utilities to convert the distributed_csr matrix to other libraries
+ */
+class CSRConversionUtilities
```

Comment:
```suggestion
class AmgclCSRConversionUtilities
```

Reply:
my original idea was to use this class also for other conversion utilities, say to mumps or pastix...but maybe we can put it in different files